examples/cfd/external_aerodynamics/domino_sensitivity/main.py

```diff
@@ -0,0 +1,491 @@
+                    ],
+                ],
+                dtype=np.float32,
+            )
+        ).to(self.device)
+
+    @cached_property
+    def surf_factors(self) -> torch.Tensor:
+        """
+        Computes the factors that are later used to unnormalize the surface predictions.
+
+        These are saved at training time based on statistics of the training data, and re-used at each inference call.
+        """
+        return torch.from_numpy(
+            np.array(
+                [
+                    [0.98881036, 0.00550783, 0.00854675, 0.00452144],
+                    [-2.4203062, -0.00740275, -0.00848471, -0.00448634],
+                ],
+                dtype=np.float32,
+            )
+        ).to(self.device)
+
+    def __call__(
+        self,
+        mesh: pv.PolyData,
+        stream_velocity: float = 38.889,
+        stencil_size: int = 7,
+        air_density: float = 1.205,
+    ) -> dict[str, np.ndarray]:
+        """Performs DoMINO inference on a given geometry to predict aerodynamic quantities.
+
+        This method takes a PyVista mesh representing a 3D geometry and computes the
+        aerodynamic predictions using the DoMINO model. It handles the data preprocessing,
+        model inference, and post-processing of results.
+
+        Args:
+            mesh: PyVista PolyData mesh representing the 3D geometry to analyze
+            stream_velocity: Inlet flow velocity in m/s. Defaults to 38.889 m/s.
+            stencil_size: Number of neighboring points to consider for surface calculations.
+                Defaults to 7.
+            air_density: Air density in kg/m³. Defaults to 1.205 kg/m³.
+
+        Returns:
+            dict: Dictionary containing the following keys:
+                - 'geometry_coordinates': Array of geometry point coordinates
+                - 'geometry_normal_sensitivity': Array of sensitivity values for each point
+                - 'pred_surf_pressure': Array of predicted surface pressure values [Pa]
+                - 'pred_surf_wall_shear_stress': Array of predicted wall shear stress values [τx, τy, τz] [Pa]
+                - 'aerodynamic_force': Array of total computed aerodynamic force [Fx, Fy, Fz] [N]
+
+        Example:
+            >>> import pyvista as pv
+            >>> from domino_sensitivity import DoMINOInference
+            >>>
+            >>> # Load geometry
+            >>> mesh = pv.read("car.stl")
+            >>>
+            >>> # Initialize inference
+            >>> domino = DoMINOInference(cfg)
+            >>>
+            >>> # Run inference
+            >>> results = domino(
+            ...     mesh=mesh,
+            ...     stream_velocity=30.0,
+            ...     stencil_size=7,
+            ...     air_density=1.205
+            ... )
+            >>>
+            >>> # Access results
+            >>> forces = results['aerodynamic_force']
+            >>> print(f"Drag force: {forces[0]:.2f} N")
+        """
```

Comment:
First, I really like this interface: pass in the mesh, and this takes care of loading the model, running inference, and unpacking the results back to sensible numbers.

The docstring is also good.  Minor nit: as presented, it isn't clear if can pass a second mesh.  Can you?  Or is that not the intended workflow?